File: 1000/1475/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.next();
        scanner.close();

        int[] count = new int[10];
        int max = 0;

        for (int i = 0; i < input.length(); i++) {
            count[input.charAt(i) - 48] += 1;
        }

        int sixesAndNines = count[6] + count[9];
        count[6] = sixesAndNines % 2 == 0 ? sixesAndNines / 2 : sixesAndNines / 2 + 1;
        count[9] = 0;

        for (int i = 0; i < count.length; i++) {
            max = Math.max(max, count[i]);
        }

        System.out.println(max);

    }
}

File: 1000/1475/main.py
import sys
import math

num = list(sys.stdin.readline())
count = [0 for i in range(10)]

for i in range(0, len(num) - 1):
    count[(int)(num[i])] += 1

sixAndNine = count[6] + count[9]
count[6] = math.floor(sixAndNine / 2) if sixAndNine % 2 == 0 else (math.floor(sixAndNine / 2) + 1)
count[9] = 0
print(max(count))